amoro-ams/amoro-ams-dashboard/vite.config.ts

```diff
@@ -95,24 +95,24 @@ export default defineConfig({
      * If you run the server on you local backend
      * Maybe you need to open the Proxy
      */
-    // proxy: {
-    //   '^/ams': {
-    //     // change the target to your backend server
-    //     // Such as target: 'http://127.0.0.1:xxx',
-    //     target: 'http://127.0.0.1:xxx',
-    //     changeOrigin: true,
-    //     configure(_, options) {
-    //       // configure proxy header here
-    //       options.headers = {
-    //         'Access-Control-Allow-Origin': '*',
-    //         'Access-Control-Allow-Credentials': 'true',
-    //         'Access-Control-Allow-Headers':
-    //           'Content-Type, Content-Length, Authorization, Accept, X-Requested-With , yourHeaderFeild',
-    //         'Access-Control-Allow-Methods': 'PUT,POST,GET,DELETE,OPTIONS'
-    //       }
-    //     }
-    //   }
-    // }
+     proxy: {
```

Comment:
why this proxy need to open? it just test to open it

Reply:
I will fixed that!